### DISCUSSION TITLE
Mobile long press button Focus

### DISCUSSION THREAD

**Discussion by ghost** [adobe/react-spectrum#7143] [General]
Hi guys, how I can remove that blue background from the tabs ? It also happens on all buttons... tried with some CSS but still nothing.

![image-20240912-145328](https://github.com/user-attachments/assets/288bccab-a31e-467f-9632-2c3ba0a0a7fd)

**Comment by reidbarber** [comment]
You can open Chrome Dev Tools and check the Computed tab, then search for background-color, then click the arrow next to it to see where that style is coming from.

**Reply by snowystinger** [reply]
Can you reproduce this in a codesandbox or stackblitz? otherwise it's hard to tell what's going on from here

**Comment by kydecker** [comment]
I'm guessing this is happening on iOS? If so, you should be able to remove the blue box with [`-webkit-tap-highlight-color`](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-tap-highlight-color):

```css
-webkit-tap-highlight-color: transparent;
```
